crates/sui-core/src/authority.rs

```diff
@@ -1632,7 +1632,7 @@ impl ExecutionState for AuthorityState {
 
                 // TODO: This return time is not ideal.
                 // TODO [2533]: edit once integrating Narwhal reconfiguration
-                Ok((Vec::default(), None))
+                Ok(Vec::default())
```

Comment:
I know you didn't change that, but is there really a need to define `Self::Outcome = Vec<u8>` (line 1602)? Can't it be `Self::Outcome = ()`?

Reply:
We co-changed that as part of https://github.com/MystenLabs/sui/pull/3526 
The goal was to remove the second member of the tuple, reverting its addition that we needed to perform in order to update the Narwhal pointer that then brought in some of your epoch work. I don't think any of the type signature of this function is anything @joyqvq or I were intentional about, we just follower the principle of introducing minimal changes (as you do when you're doing a rebase, basically).

Feed free to change this however you want.